service/storage_service.cc

```diff
@@ -6309,6 +6326,22 @@ future<> storage_service::wait_for_normal_state_handled_on_boot() {
                  fmt_nodes_with_statuses(eps));
 }
 
+void storage_service::sync_topology_change_kind_with_group0() {
+    switch (_topology_state_machine._topology.ustate) {
+    case topology::upgrade_state::done:
+        _topology_change_kind_enabled = topology_change_kind::raft;
```

Comment:
Why hide the assignment in the function? IMO better to have `topology_change_kind upgrate_state_to_topology_op_kind(upgrade_state)`.

Reply:
Sure, I can change it.